### PR TITLE
Added new Endpoints

### PR DESCRIPTION
**Pull request by codeinternetapplications**
Added Employee
Added ProjectInvoiceTerm
Added ProjectWBS
Added TimeTransaction
Updated WebhookSubscription

### CODE REVIEW

**Comment by DannyvdSluijs** [comment]
@monkie many thanks for your pull request. Would you consider using the latest version user script (Or even the one which is pending from PR #305, see https://github.com/picqer/exact-php-client/blob/27bb979a186b9eb9b50e5764d35ebe71b5706848/userscript.js for full version), as there was some improvement to correct the type hinting of this library.
Especially since your changes to the WebHookSubscription is undoing the improvements done in #289 and reverting back to the previous version with faulty type hints.

**Comment by codeinternetapplications** [comment]
No problem! Thanks

File: src/Picqer/Financials/Exact/ProjectWBSByProject.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ * @property string $Project ID of project
+ * @property string $ProjectTerm ID of invoice planning
+ * @property float $PurchaseMarkupPercentage Markup percentage
+ * @property float $Revenue Revenue of the WBS.
+ * @property string $StartDate Startdate of the WBS.If it is a deliverable then StartDate is the earliest StartDate of its children
+ * @property float $TimeQuantityToAlert Alert when exceeding this time quantity
+ * @property int $Type Type: 1-Deliverable, 2-Activity, 3-Expense
+ */
+class ProjectWBSByProject
+? projectId = {
```
Comment (stephangroen) [inline src/Picqer/Financials/Exact/ProjectWBSByProject.php]: There seems to be something wrong here :-)

**Comment by stephangroen** [comment]
Could you please add test for the new entities in https://github.com/picqer/exact-php-client/blob/master/tests/EntityTest.php? It's really simple.

**Comment by codeinternetapplications** [comment]
Fixed that!

PS: The userscript.js doesn't support endpoints with a parameter too well (/api/v1/{division}/read/project/ProjectWBSByProject?projectId={Edm.Guid}), it adds the ?projectId={Edm.Guid} part to the classname and url property.

**Comment by stephangroen** [comment]
Thanks @codeinternetapplications

**Pull request merged**